notebooks/load_DSprites.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+
+# create data folder
+mkdir -p 01_data
+cd 01_data
+
+git clone https://github.com/deepmind/dsprites-dataset.git
```

Comment:
you don't need a `.sh` for that. you should be able to directly download from python wirh `urllib` (or curl in bash) and url `https://github.com/deepmind/dsprites-dataset/blob/master/dsprites_ndarray_co1sh3sc6or40x32y32_64x64.npz?raw=true`
instead of duplicating the whole github